services/text_to_speech/v1.js

```diff
@@ -54,28 +54,36 @@ module.exports = function(RED) {
   RED.httpAdmin.get('/watson-text-to-speech/voices', function (req, res) {
     var tts = ttsutils.initTTSService(req, sApikey, sUsername, sPassword, sEndpoint);
 
-    tts.listVoices({}, function(err, voices){
-      if (err) {
+    tts.listVoices({})
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 8 spaces but found 4.](https://app.codacy.com/manual/BetaWorks-NodeRED-Watson/node-red-node-watson/pullRequest?prid=4764899)